src/Host/Client/Impl/Host/BrokerConnectionInfo.cs

```diff
@@ -4,18 +4,21 @@
 using System;
 using System.Diagnostics;
 using Microsoft.Common.Core;
+using Microsoft.Common.Core.Security;
 
 namespace Microsoft.R.Host.Client.Host {
     [DebuggerDisplay("{Uri}, IsRemote={IsRemote}, InterpreterId={InterpreterId}")]
     public struct BrokerConnectionInfo {
+        public string Name { get; }
```

Comment:
Do you use this property anywhere outside this class, or is it just required for `CredentialAuthority`?